### PR TITLE
Fix empty result

### PR DESCRIPTION
**Pull request by minhdanh**
Should fix search result empty https://github.com/dblock/slack-google-bot/issues/3

### CODE REVIEW

**Comment by dblock** [comment]
Thanks! You do need to fix the tests, please. And update CHANGELOG.

**Comment by minhdanh** [comment]
For the test to pass `GOOGLE_API_KEY` and `GOOGLE_CSE_ID` env vars are needed.

**Comment by dblock** [comment]
Shouldn't be needed. You can record a scenario with VCR and then remove any key information from it, then the test runs without making actual HTTP requests.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### 0.1.1 (08/12/2016)
+
+* Fix [#3](https://github.com/dblock/slack-google-bot/issues/3)
```
Comment (dblock) [inline CHANGELOG.md:3]: Maybe describe what this is? Like so:

```
* [#3](https://github.com/dblock/slack-google-bot/issues/3): Use a non-deprecated Google Search API - [@minhdanh](https://github.com/minhdanh).
```

**Comment by minhdanh** [comment]
I'm not sure how to do that. Can you be more specific?

**Comment by dblock** [comment]
VCR is https://github.com/vcr/vcr. It's already used here. Their doc is pretty good and try starting there.

Look at the current test, https://github.com/dblock/slack-google-bot/blob/master/spec/slack-google-bot/commands/google_spec.rb#L3. It uses a cassette, which is https://github.com/dblock/slack-google-bot/blob/master/spec/fixtures/google_search_popular_artists.yml which was recorded by running the test the first time with all the right environment variables. The second time you ran it it would just replay the response from the cassette instead of making real HTTP requests.

If you can't figure it out, I'll finish it, but give it a shot!

**Comment by dblock** [comment]
Replaced by https://github.com/dblock/slack-google-bot/pull/5.

**Pull request closed without merging**